src/main/scala/lms/transformation/mpi_nccl_transformation/distributedTensor2MPINCCLBase.scala

```diff
@@ -121,6 +121,7 @@ abstract class DistributeTensor2MPI_NCCLBase extends Transformer with MPIOps wit
   def myCUDNNComm(implicit __pos: SourceContext) = TOP(Unwrap(myCUDNNCommRep), manifest[cudnnHandleT])
 
   var cudnnTensor2Desc: HashMap[Backend.Sym, (TOP, String)] = HashMap()
+  var cudnnConv2Desc: HashMap[Seq[Int], TOP] = HashMap()
```

Comment:
maybe HashMap[Seq[Int], CUDNN_CONV_DESCRIPTOR]` is better

Reply:
I agree

Reply:
fixed in c8c4d89